packages/apps/job-launcher/client/src/pages/Job/JobDetail/index.tsx

```diff
@@ -162,7 +156,7 @@ export default function JobDetail() {
           </Grid>
         </Grid>
       </Box>
-      <Box sx={{ mt: 10 }}>
+      {/* <Box sx={{ mt: 10 }}>
```

Comment:
If we just comment this out for future usage, please add a TODO comment, otherwise, remove it.